contracts/contracts/layer_1/interfaces/clearing/IClearing.sol

```diff
@@ -254,25 +261,30 @@ interface IClearing {
         uint256 expirationTimestamp;
         address destination;
         bytes data;
-        bytes operatorData;
+        OperatorType operatorType;
     }
 
     struct ClearingRedeemData {
         uint256 amount;
         uint256 expirationTimestamp;
         bytes data;
-        bytes operatorData;
+        OperatorType operatorType;
     }
 
     struct ClearingHoldCreationData {
         uint256 amount;
         uint256 expirationTimestamp;
         bytes data;
-        bytes operatorData;
         address holdEscrow;
         uint256 holdExpirationTimestamp;
         address holdTo;
         bytes holdData;
+        OperatorType operatorType;
+    }
+
+    struct ClearingOperator {
+        bytes operatorData;
```

Comment:
Why are you introducing it? 

In my mind, operatorData is not related with thirdPartyAddress at all.